activation/wire/malfeasance.go

```diff
@@ -0,0 +1,44 @@
+package wire
+
+import (
+	"github.com/spacemeshos/go-spacemesh/common/types"
+	"github.com/spacemeshos/go-spacemesh/signing"
+)
+
+//go:generate scalegen
+
+// ProofType is an identifier for the type of proof that is encoded in the ATXProof.
+type ProofType byte
+
+const (
+	DoublePublish ProofType = iota + 1
+	DoubleMarry
+)
+
+type ATXProof struct {
+	// ProofType is the type of proof that is being provided.
+	ProofType ProofType
+	// Certificates is a slice of marriage certificates showing which identities belong to the same marriage set as
+	// the one proven to be malfeasant. Up to 1024 can be put into a single proof, since by repeatedly marrying other
+	// identities there can be much more than 256 in a malfeasant marriage set. Beyond that a second proof could be
+	// provided to show that additional identities are part of the same malfeasant marriage set.
+	Certificates []ProofCertificate `scale:"max=1024"`
```

Comment:
Maybe name it 
```suggestion
	Certificates []MarriageCertificate `scale:"max=1024"`
```

Reply:
`MarriageCertificate` already exists as a type in the `wire` package (for v2 ATXs) but this is a different type it not only contains the signature and the payload but also the signer (which isn't needed for ATXs).